### ISSUE TITLE
SCODE Docker container failing to build

### ISSUE THREAD

**Issue by ibebio** [Murali-group/Beeline#40]
Hello,

I encountered the following error when building the SCODE docker container:
```
Some packages could not be installed. This may mean that you have                                                                  
requested an impossible situation or if you are using the unstable                                                                 
distribution that some required packages have not yet been created                                                                 
or been moved out of Incoming.                                                                                                     
The following information may help to resolve the situation:                                                                       
                                                                                                                                   
The following packages have unmet dependencies:                                                                                    
 libc6-dev : Breaks: libgcc-8-dev (< 8.4.0-2~) but 8.3.0-3 is to be installed                                                      
E: Error, pkgProblemResolver::Resolve generated breaks, this may be caused by held packages.
```

To resolve it, I had to modify Dockerfile to install `gcc-8-base` after running `apt-get update`

**Comment by Hsu-Che-Wei** [comment]
Hello, 

Same problem when trying to build SCNS Docker container.
But this time, installing gcc-8-base after running apt-get update does not solve the problem.
Do you have any idea?

**Comment by tmmurali** [comment]
@Hsu-Che-Wei can you share the error you get?

**Comment by Hsu-Che-Wei** [comment]
@tmmurali 

(base) raphael830102@beeline-cwh:~/Beeline$ . ./initialize.sh 
This may take a while...
sha256:4344a80c7f38d8da9cec73a03c372272f5e6d32a904ba7383ea1a396092e656a
Docker container for ARBORETO is built and tagged as arboreto:base
sha256:367feaae8ca9b603a43d65922960d0c39455517d96e799530d3154b2dcf584d4
Docker container for GRISLI is built and tagged as grisli:base
sha256:55f529fe30035e67a2d82584faa2db62015bc4269a277ec635fab965722a62c2
Docker container for GRNVBEM is built and tagged as  grnvbem:base
sha256:ec0faad4537412325b41c3aba32bca97f63b8b39fba70d5587a7c20deebcbca6
Docker container for JUMP3 is built and tagged as  jump3:base
sha256:2eaa0ec96fec9015eca1fa4543180086674bc2c17672f3c80e73f165435d86e4
Docker container for LEAP is built and tagged as  leap:base
sha256:da240e4e3563e3d39043caeef35c8eff9f87b8b472441bf6a387ff9e7238bb5f
Docker container for PIDC is built and tagged as pidc:base
sha256:11237a3eea66916c814cb63de5403040e1ee4f591f2e8fe2e342dd7b1612a81d
Docker container for PNI is built and tagged as pni:base
sha256:95eb7ae68608a327e9935e3abf6ade7305c7abcce48b495a7135804d611fcbbd
Docker container for PPCOR is built and tagged as ppcor:base
sha256:d7f50b91e5db5a33e2237f852431be184440b390639f75ade3a36a6f804cbacc
Docker container for SINGE is built and tagged as singe:base
sha256:11f8db0dbea60ed5eeb663634df37544acc35ebfb2297a8d072ff2252da778a3
Docker container for SCNS is built and tagged as scns:base
Sending build context to Docker daemon  2.048kB
Step 1/11 : FROM r-base:3.5.3
 ---> 62c848eeb175
Step 2/11 : LABEL maintainer = "Aditya Pratapa <adyprat@vt.edu>"
 ---> Using cache
 ---> 237e322b4cab
Step 3/11 : USER root
 ---> Using cache
 ---> 7f1a56b2d021
Step 4/11 : WORKDIR /
 ---> Using cache
 ---> 4d18d0b99acc
Step 5/11 : RUN apt-get update && apt-get install -y git
 ---> Running in 31d7696d4254
Get:1 http://deb.debian.org/debian testing InRelease [116 kB]
Get:3 http://deb.debian.org/debian testing/main amd64 Packages [7,676 kB]
Get:2 http://cdn-fastly.deb.debian.org/debian sid InRelease [146 kB]
Get:4 http://cdn-fastly.deb.debian.org/debian sid/main amd64 Packages [8,388 kB]
Fetched 16.3 MB in 3s (4,902 kB/s)
Reading package lists...
Reading package lists...
Building dependency tree...
Reading state information...
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:
The following packages have unmet dependencies:
 libc6-dev : Breaks: libgcc-8-dev (< 8.4.0-2~) but 8.3.0-3 is to be installed
E: Error, pkgProblemResolver::Resolve generated breaks, this may be caused by held packages.
The command '/bin/sh -c apt-get update && apt-get install -y git' returned a non-zero code: 100
Docker container for SCODE is built and tagged as scode:base
sha256:3d796d4a560d420dfb9a87007bef6a2dc45a684cf671310b734fac96238bdf9e
Docker container for SCRIBE is built and tagged as sincerities:base
sha256:aa23db1a591b082f38694babbd4883aed5cb47b1b355a32a88f35cead33e07ee
Docker container for SINCERITIES is built and tagged as sincerities:base

**Comment by Hsu-Che-Wei** [comment]
![Uploading Screenshot 2020-09-22 at 01.02.06.png…]()

**Comment by adyprat** [comment]
Hi @Hsu-Che-Wei,
Looks like your error is not with SCNS, but rather with SCODE docker. I fixed this in 2a5a095
Let me know if this is still an issue.
